pkg/frontend/transport/handler.go

```diff
@@ -116,6 +117,11 @@ func NewHandler(cfg HandlerConfig, roundTripper http.RoundTripper, log log.Logge
 			Help: "Number of series fetched to execute a query.",
 		}, []string{"user"})
 
+		h.querySamples = promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
+			Name: "cortex_query_samples_total",
+			Help: "Number of samples to execute a query.",
```

Comment:
`Number of samples fetched`

Reply:
Fixed it. Thanks for the review. @yeya24